CmdStarter.Tests.Commands/Attributes/AutoComplete/MultiAutoComplete.cs

```diff
@@ -0,0 +1,43 @@
+﻿using com.cyberinternauts.csharp.CmdStarter.Tests.Commands.Interfaces;
+using System.CommandLine;
+using System.CommandLine.Completions;
+
+namespace com.cyberinternauts.csharp.CmdStarter.Tests.Commands.Attributes.AutoComplete
+{
+    public sealed class MultiAutoComplete : StarterCommand, IHasAutoComplete
```

Comment:
To be consistent with all the `Multi(Single|Multi)` classes, maybe shall name this one `SingleMulti` and `Single` into `SingleSingle`?

Reply:
Done.